lib/ansible/modules/set_fact.py

```diff
@@ -28,7 +28,8 @@
     required: true
   cacheable:
     description:
-      - This boolean converts the variable into an actual 'fact' which will also be added to the fact cache, if fact caching is enabled.
+      - This boolean converts the variable into an actual 'fact' which will also be added to the fact cache
+        It does not enable fact caching across runs, it just means it will work with it if already enabled.
```

Comment:
```suggestion
      - This boolean converts the variable into an actual 'fact' which will also be added to the fact cache.
        It does not enable fact caching across runs, it just means it will work with it if already enabled.
```